include/k4a/k4a.h

```diff
@@ -886,13 +947,76 @@ K4A_EXPORT uint32_t k4a_image_get_white_balance(k4a_image_t image_handle);
  */
 K4A_EXPORT uint32_t k4a_image_get_iso_speed(k4a_image_t image_handle);
 
-/** Set the time stamp, in microseconds, of the image.
+/** Set the device time stamp, in microseconds, of the image.
+ *
+ * \param image_handle
+ * Handle of the image to set the timestamp on.
+ *
+ * \param timestamp_usec
+ * Device timestamp of the image in microseconds.
+ *
+ * \remarks
+ * Use this function in conjunction with k4a_image_create() or k4a_image_create_from_buffer() to construct a
+ * \ref k4a_image_t.
+ *
+ * \remarks
+ * Set a timestamp of 0 to indicate that the timestamp is not valid.
```

Comment:
In `k4a_image_get_device_timestamp_usec` we mention that a timestamp of 0 is valid. Here we mention that its not valid. Is there a difference between a created frame and a recorded frame?